src/main/java/genji/Genji.java

```diff
@@ -0,0 +1,39 @@
+package genji;
+
+import genji.command.Command;
+import genji.task.TaskList;
+
+public class Genji {
```

Comment:
Remember to add a class description header for JavaDocs to describe briefly what the class is.